vehicles.ini

```diff
@@ -73,6 +73,7 @@ stockcars toyotacamry,NASCAR Cup Toyota Camry,4,9000
 stockcars fordthunderbird87,NASCAR Ford Thunderbird - 1987,4,7000
 stockcars chevycamarozl12022,NASCAR Cup Next Gen Chevrolet Camaro ZL1,5,9000
 stockcars fordmustang2022,NASCAR Cup Next Gen Ford Mustang,5,9000
+stockcars chevycamarozl12022,NASCAR Cup Next Gen Chevrolet Camaro ZL1,5,9000
```

Comment:
Incorrect entry. Correctr entry is;
stockcars toyotacamry2022,NASCAR Cup Next Gen Toyota Camry,5,9000